pkg/oidcclient/login.go

```diff
@@ -445,8 +656,11 @@ func (h *handlerState) handleAuthCodeCallback(w http.ResponseWriter, r *http.Req
 		return httperr.New(http.StatusForbidden, "missing or invalid state parameter")
 	}
 
-	// Check for error response parameters.
+	// Check for error response parameters. See https://openid.net/specs/openid-connect-core-1_0.html#AuthError.
 	if errorParam := params.Get("error"); errorParam != "" {
+		if errorDescParam := params.Get("error_description"); errorDescParam != "" {
```

Comment:
I remember writing a bit of code like this and then deleting it. I'm fine leaving it but it seems like `error_description` is meant to be for "application developers" rather than "end users": https://www.oauth.com/oauth2-servers/server-side-apps/possible-errors/

Reply:
That's an interesting interpretation of the spec, which says:

> "Human-readable ASCII encoded text description of the error."

It doesn't say _which_ human should be able to read it, i.e. developer vs. end user. 😆

I can see how a properly internationalized client, like a web app, would not want to use the ASCII (non-UTF) description and would prefer to make up their own descriptions in each supported locale based on the error code. For us, we are not an internationalized client, so it feels like giving the user more information about why their auth failed would be better than giving them less information, no?

Reply:
Yeah I think it's a net positive, just thought it was kind of curious that other people had interpreted the spec that way.